File: JustRentItAPI/JustRentItAPI/Models/DTOs/UserDTO.cs
using JustRentItAPI.Models.Enums;

namespace JustRentItAPI.Models.DTOs
{
    public class UserDTO
    {
        public int UserID { get; set; }
        public string FirstName { get; set; } = string.Empty;
        public string LastName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public UserRole Role { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdateAt { get; set; }
        public List<int> WishlistDressIds { get; set; } = new();
    }
}

File: just-rent-it-app/src/models/DTOs/UserDTO.ts
import { UserRole } from "../Enums/UserRole";

export interface UserDTO {
  userID: number;
  firstName: string;
  lastName: string;
  email: string;
  phone: string;
  role: UserRole;
  createdDate: string;
  updateAt: string;
  wishlistDressIds: number[];
}
